### PR TITLE
Update apps-integrations.md

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to update the installing apps as per our new UI.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Update docs for app catalog refresh](https://sumologic.atlassian.net/browse/SUMO-234406)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

File: docs/get-started/apps-integrations.md [hunk 1]
```diff
@@ -37,10 +39,18 @@ What makes Next-Gen different from Classic apps?
 * Upon app installation, content is installed into a shared **Installed Apps** folder that's visible to your entire org, making it easier to locate and upgrade, when needed.
 * Installed app content is immutable, which ensure that any apps upgrades do not overwrite any customizations you may apply. If you wish to customize the content, you'll need to first make a custom copy of the content.
 
+import AppInstall from '../reuse/apps/app-install-v2.md';
```
Comment (kimsauce) [inline docs/get-started/apps-integrations.md:42]:
```suggestion

#### How to install Next-Gen Apps

import AppInstall from '../reuse/apps/app-install-v2.md';
```

File: docs/get-started/apps-integrations.md [hunk 2]
```diff
@@ -37,10 +39,18 @@ What makes Next-Gen different from Classic apps?
...
+
+<AppInstall/>
+
 ### Classic Apps (Legacy)
 
 All of our existing Classic apps will be converted to Next-Gen. In the meantime, our Classic apps will remain available to you within the Application Catalog.
 
+import ClassicAppInstall from '../reuse/apps/app-install.md';
+
+<ClassicAppInstall/>
```
Comment (kimsauce) [inline docs/get-started/apps-integrations.md:52]:
```suggestion
#### How to install Classic Apps

import ClassicAppInstall from '../reuse/apps/app-install.md';

<ClassicAppInstall/>
```

**Review by kimsauce — DISMISSED** [review]
Adding the each header would help. Otherwise it'd go straight into the content -  

Screenshot 2024-03-29 at 10 33 30 AM

I've added these as Suggestions to the draft.

**Comment by JV0812** [comment]
Thanks @kimsauce, this looks good now.

**Pull request merged**